server/src/internalClusterTest/java/org/opensearch/search/nested/SimpleNestedIT.java

```diff
@@ -500,6 +499,98 @@ public void testExplain() throws Exception {
         assertThat(explanation.toString(), startsWith("0.36464313 = Score based on 2 child docs in range from 0 to 1"));
     }
 
+    public void testExplainMultipleDocIndexingNoDeletedDoc() throws Exception {
+        assertAcked(
+            prepareCreate("test").setMapping(
+                jsonBuilder().startObject()
+                    .startObject("properties")
+                    .startObject("nested1")
+                    .field("type", "nested")
+                    .endObject()
+                    .endObject()
+                    .endObject()
+            )
+        );
+
+        ensureGreen();
+
+        for (int i = 0; i < 5; i++) {
+            client().prepareIndex("test")
+                .setId(String.valueOf(i))
+                .setSource(
+                    jsonBuilder().startObject()
+                        .field("field" + i, "value" + i)
```

Comment:
nit: shouldn't only the value change change in the loop?

Reply:
updated the mappings above to make it clearer.